components/channel_header_dropdown/channel_header_dropdown_items.js

```diff
@@ -267,7 +283,9 @@ export default class ChannelHeaderDropdown extends React.PureComponent {
                         />
                     </ChannelPermissionGate>
                 </Menu.Group>
-
+                <Menu.Group>
```

Comment:
Will there be a divider between normal options and plugin options?

Reply:
Yes, `Menu.Group` includes the divider. See attached screenshot in the PR description.